include/SDL3/SDL_audio.h

```diff
@@ -705,7 +705,7 @@ typedef struct SDL_AudioStream SDL_AudioStream;
  * \param dst_format The format of the desired audio output
  * \param dst_channels The number of channels of the desired audio output
  * \param dst_rate The sampling rate of the desired audio output
- * \returns 0 on success, or -1 on error.
+ * \returns 0 on success, or a negative error code.
```

Comment:
Can we change these to the standard verbiage?
```
 * \returns 0 on success or a negative error code on failure; call
 *          SDL_GetError() for more information.
```

Reply:
yes, I'll try later. do you think if it this is put on 1 line (eg with sed)? this is going to get replaced correctly by the bot ?

Reply:
Yes, it should be.